app/src/services/email.ts

```diff
@@ -79,6 +97,28 @@ const service = {
     }
   },
 
+  /**
+   * @function logEmail
+   * Logs CHES email api calls
+   * @param {Email_data | null} data Object containing CHES response, or null on error
+   * @param {Array<string>} recipients Array of email strings
+   * @param {status} status Http status of CHES response
+   * @returns null
+   */
+  logEmail: async (data: Email_data | null, recipients: Array<string>, status: number) => {
+    return await prisma.$transaction(async (trx) => {
+      return await trx.email_log.createMany({
+        data: recipients.map((x) => ({
+          email_id: uuidv4(),
+          msg_id: data?.messages?.[0].msgId ?? null,
```

Comment:
the `?? null` is redundant as your make this operation in the the `toPrismaModel` function

Reply:
Fixed.